CONTRIBUTING.md

```diff
@@ -148,7 +148,7 @@ Note that adjusting the `schema.json` files will also affect the CLI manuals and
 To run `nx-dev` locally, run the command:
 
 ```bash
-npx nx serve nx-dev
+yarn dlx nx serve nx-dev
```

Comment:
`dlx` will **d**own**l**oad-and-e**x**ecute `nx` from the registry, that's probably not what you want here

Reply:
That's what `npx` did as well. I personally don't see a problem with it, but understand your point.

Reply:
From npx's documentation:

> Executes `<command>` either from a local `node_modules/.bin`, or from a central cache, installing any packages needed in order for `<command>` to run.

So actually it looks for a local version first.